demos/ASP.Net Core API Gateway/Divergent.Sales.ViewModelComposition/OrdersListViewModelAppender.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Divergent.Sales.ViewModelComposition.Events;
+using ITOps.ViewModelComposition;
+using ITOps.ViewModelComposition.Json;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Divergent.Sales.ViewModelComposition
+{
+    public class OrdersListViewModelAppender : IViewModelAppender
+    {
+        public bool Matches(RouteData routeData, string verb)
```

Comment:
yes and no, probably in a production ready thing would be a nice to have, in a sample not necessarily.